cmd/misc.go

```diff
@@ -76,12 +78,38 @@ var getblobberscmd = &cobra.Command{
 		if err != nil {
 			log.Fatal(err)
 		}
-		blobbers, err := zcncore.GetBlobbers(!active)
-		if err == nil {
-			printBlobberList(blobbers)
-		} else {
-			ExitWithError("\nERROR: Get blobbers failed. " + err.Error() + "\n")
+		wg := &sync.WaitGroup{}
+		statusBar := &ZCNStatus{wg: wg}
+
+		var blobberList []*sdk.Blobber
+		limit, offset := 20, 0
+
+		for {
+			wg.Add(1)
+			zcncore.GetBlobbers(statusBar, limit, offset, !active)
+			wg.Wait()
+
+			type nodes struct {
+				Nodes []*sdk.Blobber
+			}
+
+			var wrap nodes
+
+			err := json.Unmarshal([]byte(statusBar.errMsg), &wrap)
```

Comment:
Why the results are in the `statusBar.errMsg`, shouldn't it be returned by the `GetBlobbers()` directly.

Reply:
Well, yes, I will address this is in a separate. This refactoring was for pagination handling